<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ntu_addcourse</class>
 <widget class="QWidget" name="ntu_addcourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>621</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>412</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#year_error_mess, #error_mess {
	color: red;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mainBodyContainer" native="true">
     <property name="styleSheet">
      <string notr="true">#mainBodyContainer {
	background-color: #fafafa;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QPushButton" name="close_btn">
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/assets/icons/Close.png</normaloff>:/icons/assets/icons/Close.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Tên học phần</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="courseName_le_2">
              <property name="minimumSize">
               <size>
                <width>160</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1200</width>
                <height>32</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>120</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>120</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>12</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="horizontalSpacing">
             <number>12</number>
            </property>
            <property name="verticalSpacing">
             <number>28</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Năm học:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="year_le">
              <property name="minimumSize">
               <size>
                <width>110</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>140</width>
                <height>32</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="inputMask">
               <string>xxxx-xxxx</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Học kỳ:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="semester_cb">
              <property name="minimumSize">
               <size>
                <width>110</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>140</width>
                <height>32</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="year_error_mess">
              <property name="minimumSize">
               <size>
                <width>104</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>104</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>140</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>140</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>12</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="horizontalSpacing">
             <number>12</number>
            </property>
            <property name="verticalSpacing">
             <number>6</number>
            </property>
            <item row="0" column="1">
             <widget class="QLineEdit" name="folder_le">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="selectFolder_btn">
              <property name="minimumSize">
               <size>
                <width>36</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>36</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Chọn thư mục</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/icons/assets/icons/Opened Folder.png</normaloff>:/icons/assets/icons/Opened Folder.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Website:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Thư mục:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="web_le">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="warming_mess">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>true</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="error_mess">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>12</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignRight|Qt::AlignBottom">
             <widget class="QPushButton" name="save_btn">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Lưu</string>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
